hip_data_tools/aws/s3.py

```diff
@@ -372,35 +372,6 @@ def move_recursive(self,
         if delete_after_copy:
             self.delete_recursive(source_dir)
 
-    def move_recursive_to_different_bucket(self, source_key_prefix, destination_bucket_name,
```

Comment:
We have used this function for the braze DAG. We need to change that when we update the hip data tools version.

Reply:
yep